java/README.md

```diff
@@ -72,6 +72,35 @@ find java/format/src -type f | while read file; do
 done
 ```
 
+# add module-info.java to the generated code
+
+Make sure to add the `java/format/src/main/java/module-format.java` as follows.
```

Comment:
Er, wouldn't you just revert the deletion via Git?

Reply:
Ahh 😌 we could sure.

Reply:
I will remove that.